src/scf-release/jobs/acceptance-tests-brain/templates/run.erb

```diff
@@ -9,17 +9,13 @@ set -o errexit
 HELM_BIN_DIR=/var/vcap/packages/helm/bin
 HELM_VERSION="$(${HELM_BIN_DIR}/helm version | grep Server | sed -e 's/.*"v//' -e 's/".*//')"
 
-echo Packaged helm clients ...
-( cd ${HELM_BIN_DIR} ; find . -type f | sed -e 's/^/= /' )
 echo Helm server ${HELM_VERSION}
-echo Compare
 
-if test -f "${HELM_BIN_DIR}/${HELM_VERSION}/helm"
-then
-    echo Match
-else
-    echo "Helm server ${HELM_VERSION} not supported, only: $(cd ${HELM_BIN_DIR} ; ls | sort)"
-    exit 1
+if ! test -f "${HELM_BIN_DIR}/${HELM_VERSION}/helm"; then
```

Comment:
`! test` feels wrong compared to `test !` ... Checking bash manpage it looks to be valid.

Reply:
To avoid confusion, did `[ ! -f ]`.